build-plugin/src/main/kotlin/thunderbird.library.android.compose.gradle.kts

```diff
@@ -0,0 +1,25 @@
+plugins {
+    id("thunderbird.library.android")
+}
+
+android {
+    configureSharedComposeConfig(libs)
+}
+
+dependencies {
+    val composeBom = platform("androidx.compose:compose-bom:${libs.versions.androidxComposeBom.get()}")
+    implementation(composeBom)
+    androidTestImplementation(composeBom)
+
+    implementation("androidx.compose.material:material")
+
+    // Android Studio Preview support
+    implementation("androidx.compose.ui:ui-tooling-preview")
+    debugImplementation("androidx.compose.ui:ui-tooling")
+
+    // UI Tests
+    debugImplementation("androidx.compose.ui:ui-test-manifest")
+    androidTestImplementation("androidx.compose.ui:ui-test-junit4")
```

Comment:
Can we add entries for all of these to the toml file? It would be nice if looking at that file was enough to see a list of all of our (direct) app dependencies.

Reply:
For the version catalog you need to add a version, with the `compose-bom` that's not possible. This would require to manage all dependencies in the catalog instead.

Reply:
I was afraid of that :disappointed: